<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventNamesDockWidget</class>
 <widget class="QDockWidget" name="EventNamesDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LRS-Editor Event Names</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_event_class_name">
        <property name="text">
         <string>Event Class Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="txt_event_class_name">
        <property name="text">
         <string>txt_event_class_name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="le_event_name">
      <property name="toolTip">
       <string>Filter Event Names</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>241</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
